x/evm/keeper/state_transition.go

```diff
@@ -516,7 +525,9 @@ func (k *Keeper) ApplyMessageWithConfig(
 	// Ethermint original code:
 	// Logs: types.NewLogsFromEth(stateDB.Logs()),
 	var replyLog StateDBGetLogsReply
-	err = sgxRPCClient.StateDBGetLogs(StateDBGetLogsArgs{}, &replyLog)
+	err = sgxRPCClient.StateDBGetLogs(StateDBGetLogsArgs{
+		HandlerId: handlerId,
+	}, &replyLog)
 	if err != nil {
 		return nil, err
```

Comment:
I think we need a StopEVM, to clear the sdkCtxs map (and also the `evms` map on inco-monorepo). See https://github.com/Inco-fhevm/ethermint/pull/3/files#diff-feaf82fa09f2e8d80af866070cc26213ea09ec4c89ae8a122a6bb4153f8450d6R625 for the gRPC one.

Reply:
Done.